### DISCUSSION TITLE
Export the function to open the modal

### DISCUSSION THREAD

**Discussion by uguareschi** [rainbow-me/rainbowkit#573] [Ideas]
Right now if I want to open the modal I need to use the custom button, but what if I want to show the modal like a side effect of something?

**Comment by valleyofblackpanther** [comment]
@uguareschi are you talking about changing the connect wallet button's ambience or design in the examples? 
[examples](https://github.com/rainbow-me/rainbowkit/tree/main/examples) I guess you are talking about this directory.

**Comment by markdalgleish** [comment]
This is out now in [RainbowKit v0.4.3.](https://github.com/rainbow-me/rainbowkit/releases/tag/%40rainbow-me%2Frainbowkit%400.4.3)

You can check out the docs here: https://www.rainbowkit.com/docs/modal-hooks
